### PR TITLE
Removed vague and unneceessary relative sentence

### PR DESCRIPTION
**Pull request by noha**
This part of the text raised another discussion. It is vague and not helping to have that sentence. So I removed it

### CODE REVIEW

**Comment by astares** [comment]
Makes sense

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ Specifically:
 - Be kind. Be courteous. Be welcoming.
 - Listen. Consider and acknowledge people’s points before responding.
 
-If any member of the community violates this code of conduct, the maintainers of the Pharo project may take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate. We will not tolerate harassment from anyone in the Pharo community, even outside of Pharo’s public communication channels.
+If any member of the community violates this code of conduct, the maintainers of the Pharo project may take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate. We will not tolerate harassment from anyone in the Pharo community.
 
-If you are subject to or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).
\ No newline at end of file
+If you are subject to or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).
```
Comment (jgfoster) [inline CODE_OF_CONDUCT.md:14]: [To imply that there is an openness to hear both sides and that there is some due process, I invite consideration of the following:]

If you are subject to *, accused of,* or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).

Comment (estebanlm) [reply CODE_OF_CONDUCT.md:14]: I think is reasonable. 
PR please :)

**Comment by bencoman** [comment]
Already merged, but just want to record my support for this wording.  Also Jame's amendment is fine.

**Pull request merged**